packages/clay-navigation-bar/src/index.tsx

```diff
@@ -43,6 +43,15 @@ export interface IProps
 	 */
 	inverted?: boolean;
 
+	/**
+	 * Defines aria-label messages to display for the screen reader.
+	 */
+	messages: {
```

Comment:
Here we also need to make this API optional to avoid the same problem I mentioned above.

Reply:
I added the default values, let me know if it looks good @matuzalemsteles